sources/packages/backend/libs/integrations/src/cas/cas.service.ts

```diff
@@ -164,21 +165,10 @@ export class CASService {
         },
       };
     } catch (error: unknown) {
-      if (
-        error instanceof AxiosError &&
-        error.response?.status === HttpStatus.BAD_REQUEST &&
-        !!error.response?.data[CAS_RETURNED_MESSAGES]
-      ) {
-        // Checking for bad request errors for better logging while the
-        // specific ticket to handle exception is pending.
-        throw new CustomNamedError(
-          error.response.data[CAS_RETURNED_MESSAGES],
-          CAS_BAD_REQUEST,
-        );
-      }
-      throw new Error("Error while creating supplier and site on CAS.", {
-        cause: error,
-      });
+      this.handleBadRequestError(
+        error,
+        "Error while creating supplier and site on CAS.",
```

Comment:
The error message at lines 170 nd 211 are probably not intentionally the same. Can you check, please?